modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/GodotEnums.cs

```diff
@@ -120,6 +120,7 @@ internal enum PropertyUsageFlags
         EditorInstantiateObject = 33554432,
         EditorBasicSetting = 67108864,
         ReadOnly = 134217728,
+        NeverDuplicate = 1 << 28,
```

Comment:
To be consistent with the rest of the enum:

```suggestion
        NeverDuplicate = 268435456,
```

For context on why the enum is defined this way: https://github.com/godotengine/godot/pull/64089#discussion_r940810214.

Reply:
That's because it's directly copied from the generated bindings. This will be changed soon, any way.